cloudhub/modules/ROOT/pages/app-migration.adoc

```diff
@@ -57,14 +57,18 @@ The *Host* dropdown list shows only the domains added under the certificates of
 
 === Considerations and Limitations
 
-The upgrade process doesn’t automatically map some configuration settings to the new CloudHub 2.0 application. Go to your new CloudHub 2.0 application settings to finish the configuration.
+The upgrade process doesn’t automatically map some configuration settings to the new CloudHub 2.0 application. 
 
 ==== Application Properties 
+
 The upgrade process doesn't carry over application properties to CloudHub 2.0. Reconfigure these properties manually for a successful upgrade:
-* When properties are initially copied during CloudHub 2.0 application configuration, they don't persist unless you edit at least one property or add a new property. Editing a single property, or adding a new one, causes the edited or new property and all other copied properties to persist.
+
+* When the upgrade process initially copies your CloudHub app's properties during CloudHub 2.0 application configuration, they don't persist unless you edit at least one property or add a new property. Editing a single property, or adding a new one, causes the edited or new property and all other copied properties to persist.
```

Comment:
What's the reference for "they"? Are "they" the properties? If so, you may say:
* The upgrade process initially copies your CloudHub app's properties during the CloudHub 2.0 application configuration. These properties don't persist unless you edit at least one property or add a new property. Editing a single property, or adding a new one, causes the edited or new property and all other copied properties to persist.